External/FEXCore/Source/Utils/FileLoading.cpp

```diff
@@ -47,5 +49,16 @@ bool LoadFile(std::vector<char> &Data, const std::string &Filepath, size_t Fixed
   return true;
 }
 
+bool LoadFileToBuffer(const std::string &Filepath, char *Buffer, size_t FixedSize) {
```

Comment:
Can we use `std::span`, yet? Would be nicer than passing pointer+size pairs around.

Reply:
Switched over to using span.